.github/workflows/tag-version-this-repo.yml

```diff
@@ -0,0 +1,13 @@
+name: Tag Version (this repo)
+on:
+  push:
+    branches:
+      - release/**
+
+jobs:
+  run:
+    name: Tag Version Automation
+    if: github.event.pusher.name != 'LombiqBot'
```

Comment:
Good question. After the `@release/v1.0` refs are updated to `@v1.0` in the GHA files, they are automatically committed on the `release/v1.0` branch and pushed (by the `LombiqBot` user). And because the workflow is triggered on the `push` event to any `release/**` branch, it kicks off a 2nd workflow run.

This specific condition exits early from that 2nd workflow run.

One important question to ask here is, do we really want this commit on the `release/v1.0` branch and then have it pushed?

One problem I see is any additional commits pushed to the `release/v1.0` branch, fixing a small bug for example, may be challenging. I think you'll get an out of date error when pushing and you'll need to pull the latest commit, All the refs will point to the new `@v1.0` tag instead of the `@release/v1.0` branch. So you'll have to change them all back before pushing again.

Maybe the solution is to put the commit on a different branch like `main` or a new one like `release-asset/v1.0`. Then at the very least, the 2nd run of the workflow won't be triggered on the push to the `release/v1.0` branch.

Reply:
> And because the workflow is triggered on the push event to any release/** branch, it kicks off a 2nd workflow run.

Yep, that makes sense the we need to prevent it from re-triggering itself.

> One problem I see is any additional commits pushed to the release/v1.0 branch, fixing a small bug for example, may be challenging.

That's fine, if a subsequent fix need to be released after it, then that will be in a PR that targets the default branch and then we can start the release process from that PR's merge commit. The release branches can remain "floating", Microsoft does the same in the https://github.com/actions/runner-images repo for example.